nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -70,6 +63,12 @@ let
           ${xorg.xrdb}/bin/xrdb -merge ~/.Xdefaults
       fi
 
+      # Import environment variables into the systemd user environment.
+      ${optionalString (cfg.displayManager.importedVariables != []) (
+        "${config.systemd.package}/bin/systemctl --user import-environment "
```

Comment:
This should be ` /run/current-system/systemd/bin/systemctl` - we always want to signal systemctl from the currently running systemd, not whatever systemd the (now switched-to) configuration should be running with.

Also see https://github.com/NixOS/nixpkgs/pull/88492 which originally introduced it like that.

Reply:
I see, sorry about that. Fixed.